src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        return Optional.of(carDao.get(id).orElseThrow());
```

Comment:
```suggestion
        return Optional.of(carDao.get(id).orElseThrow(
        () -> new NoSuchElementException("Can't get car by this id: " + id)));
```
better add the exception and explanation why exception appeared